eng/pipelines/build.yml

```diff
@@ -145,6 +156,7 @@ stages:
             vmImage: ubuntu-20.04
           variables:
             AutorestCSharpVersion: $[stageDependencies.Build_and_Test.Build.outputs['Package.AutorestCSharpVersion']]
+            CadlEmitterVersion: $[stageDependencies.Build_and_Test.Build.outputs['Publish.CadlEmitterVersion']]
```

Comment:
Where is to define CadlEmitterVersion? it is better that the minVersion of caldEmitterVersion is the same as autorest.csharp. such as 20221202.1

Reply:
@chunyu3 It's defined in [PublishRelease.ps1](https://github.com/Azure/autorest.csharp/pull/2914/files#diff-3a01bf31f85915a7085de14ff3ccce2836de59f0acf8a98696edfa9d545580d2R53):

> Write-Host "##vso[task.setvariable variable=CadlEmitterVersion;isoutput=true]$devVersion"

Its suffix is the same as `autorest.csharp`, e.g. `beta.20211202.1`, see the updated [csharp-emitter-package.json](https://github.com/Azure/azure-sdk-for-net/pull/32866/files#diff-5e7d37047100b2bc3a83b37f06d0bb6076340a33e6479168b309de300a5e5395):

> "@azure-tools/cadl-csharp": "0.1.10-beta.20221204.2"